### DISCUSSION TITLE
Working on amsmath LaTeX examples

### DISCUSSION THREAD

**Discussion by eeholmes** [quarto-dev/quarto-cli#10545] [Show and Tell]
### Description

I have started a repo to plug away at the issues around using amsmath LaTeX in Quarto books: https://github.com/nmfs-opensci/quarto-amsmath. It's related to this issue around getting numbered equations to work in html output in Quarto books:
https://github.com/quarto-dev/quarto-cli/issues/2275

Note for those wanting some examples of numbered equations and fancy math. I have examples here, but generally speaking Pandoc/MathJax understands most of the amsmath stuff.
* https://nmfs-opensci.github.io/quarto-amsmath/numbered_equations.html
* https://nmfs-opensci.github.io/quarto-amsmath/fancy_equations.html

Note this is narrowly focused on html and PDF rendering of pure amsmath LaTeX equations and cross-refs in Quarto books.

fyi @mcanouil @cscheid

**Comment by cscheid** [comment]
Thank you - this is super valuable. 

(I'm sorry we ended up not meeting at posit::conf, but please continue to post issues and be in touch!) We'd love to reduce this kind of friction in the future for everyone. Unfortunately for you (and very appreciatedly by us!), the best way for us to learn about the snags is for someone to work through them in a particular context!
